scripts/amend_pbn/amend_pbn.py

```diff
@@ -0,0 +1,39 @@
+from inspirehep.curation.search_check_do import SearchCheckDo
+
+tag_info = ("parent_isbn", "9789811947506")
+
+missing_info = {
+    "journal_title": "Springer Proc.Math.Stat.",
+    "journal_volume": "396",
+    "parent_record": {"$ref": "https://inspirehep.net/api/literature/2628642"},
+}
+
+
+class AmendPBN(SearchCheckDo):
+    """Add missing info to PBNs with tag"""
+
+    query = "publication_info.%s:%s" % tag_info
+
+    @staticmethod
+    def check(record, logger, state):
+        # flag PBN containing tag info
+
+        state["pos_tag"] = []
+        for npbn, pbn in enumerate(record.get("publication_info", [])):
+            tag = pbn.get(tag_info(0), "")
+            if tag == tag_info(1):
```

Comment:
`tag_info` is not a function but a tuple
```suggestion
            tag = pbn.get(tag_info[0], "")
            if tag == tag_info[1]:
```